elements/bulbs-video/components/revealed.js

```diff
@@ -202,7 +202,7 @@ export default class Revealed extends React.Component {
   }
 
   makeVideoPlayer (element, videoMeta) {
-    element.id = jwPlayerIdCounter++;
+    element.id = `jw-player-${jwPlayerIdCounter++}`;
```

Comment:
@MelizzaP, can you confirm this also increments the original value?  If there are 3 players on the same page, I wanted to confirm the assignment works.  Maybe we can point a test link at Collin's american voters page as a perfect recent example. 

e.g. 
```
let jwPlayerIdCounter=0;
foo = `jw-player-${jwPlayerIdCounter++}`;
// value foo is `jw-player-1`, but is jwPlayerIdCounter the variable still 0?
```